CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+## v0.0.33-alhpa
+* Move `browserSetup` function call to constructor phase in hot-index lifecycle.
```

Comment:
could you please explain 'what' not 'how'? like 'Fix race condition for browserSetup being called after rendering styleguide'

Reply:
could you please update package.json as well?

Reply:
yes, I added update in this pull request.